packages/backend/src/app.ts

```diff
@@ -403,17 +405,26 @@ export class App {
     const { resourceServerSpec, walletAddressServerSpec } =
       await this.container.use('openApi')
 
+    const validatorMiddlewareOptions = {
+      validateRequest: true,
+      validateResponse: process.env.NODE_ENV !== 'production'
```

Comment:
This is what the `createValidatorMiddleware` function was doing previously, now to keep the same behaviour, it has to be stated explicitly 

https://github.com/interledger/open-payments/releases/tag/%40interledger%2Fopenapi%402.0.0